shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/FeaturePromo.kt

```diff
@@ -2,7 +2,8 @@ package com.mbta.tid.mbta_app.model
 
 enum class FeaturePromo(val addedInAndroidVersion: AppVersion, val addedInIosVersion: AppVersion) {
     // ktfmt does not like empty enums as of the version we get with spotless
-    Nothing(AppVersion(0u, 0u, 0u));
+    Nothing(AppVersion(0u, 0u, 0u)),
+    CombinedStopAndTrip(AppVersion(0u, 0u, 0u), AppVersion(1u, 2u, 0u));
```

Comment:
Now that there's a real feature in the list, we shouldn't need `Nothing` or the comment.